snowflake_blueprints/upload_file.py

```diff
@@ -1,108 +1,110 @@
-from sqlalchemy.exc import DatabaseError, DBAPIError, ProgrammingError
+import contextlib
 import pandas as pd
-from sqlalchemy import create_engine
-from snowflake.sqlalchemy import URL
 import argparse
-import re
 import sys
+import re
 import shipyard_utils as shipyard
-from snowflake.connector.pandas_tools import pd_writer, write_pandas
 import dask.dataframe as dd
-import snowflake.connector
 import ast
-import snowflake.sqlalchemy as sql
-from sqlalchemy import Table, Column, Integer, String, MetaData
+import os
+
+from sqlalchemy.exc import DatabaseError, DBAPIError, ProgrammingError
+from sqlalchemy import create_engine
+from snowflake.sqlalchemy import URL
+
 from copy import deepcopy
+
```

Comment:
Removed unused modules and organized the imports to that imports are grouped to be consistent with our mono repo styling